src/helpers/DirHelper.ts

```diff
@@ -0,0 +1,23 @@
+import * as fs from "fs";
+import * as path from "path";
+
+export class DirHelper {
+    public static mkdirIfNotExist(dir: string) {
+        if (!this.exists(dir)) {
```

Comment:
You can bail here early to avoid having the whole function code nested inside a if. 

```ts
if (this.exists(dir)) {
    return;
}

// everything else
```